### PR TITLE
add notes on github project settings

### PR DESCRIPTION
**Pull request by gilesknap**

### CODE REVIEW

File: docs/how-to/github-settings.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    - tick: Require a pull request before merging
+    - un-tick: Require approvals
+    - tick: Include administrators
+
+  - skeleton
+
+    - tick: Require a pull request before merging
+    - un-tick: Require approvals
+    - tick: Require linear history
+    - tick: Include administrators
```
Comment (coretl) [inline docs/how-to/github-settings.rst]: We don't have a skeleton branch in skeleton-bsed projects...

File: docs/how-to/github-settings.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  - Enable Dependabot alerts, Dependabot security updates
+
+- General
+
+  - Features
+
+    - tick: Issues
+  - Pull Requests
+    - un-tick: Allow merge commits
+    - un-tick: Allow squash merging
```
Comment (coretl) [inline docs/how-to/github-settings.rst]: again, only in skeleton

**Review by coretl — CHANGES_REQUESTED** [review]

File: docs/how-to/github-settings.rst (hunk 1 above)
Comment (gilesknap) [reply docs/how-to/github-settings.rst]: right - I only worked this out this morning when updating ibek. removed.

File: docs/how-to/github-settings.rst (hunk 2 above)
Comment (gilesknap) [reply docs/how-to/github-settings.rst]: removed.

**Pull request closed without merging**